bedrock/security/views.py

```diff
@@ -205,19 +206,26 @@ class KVRedirectsView(CachedRedirectView):
     prod_ver_re = re.compile('(\w+)(\d{2})$')
 
     def get_redirect_url(self, *args, **kwargs):
-        url_component = kwargs['filename']
+        url_component = kwargs['filename'].replace(' ', '')
+        try:
+            return reverse(**self.get_redirect_args(url_component))
+        except NoReverseMatch:
+            return None
+
+    def get_redirect_args(self, url_component):
         if url_component == 'suite17':
-            return reverse('security.product-advisories', kwargs={'slug': 'mozilla-suite'})
+            return dict(viewname='security.product-advisories',
```

Comment:
yeah, mostly did it to avoid a larger diff, also because nested literal dicts aren't very readable IMHO. But I'd be happy either way really.